### ISSUE TITLE
MockUtil.resetMock() doesn't create InvocationNotifierHandler when recreating a filter

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dlsgusrn7577/mockito#282]
```
What steps will reproduce the problem?
1. create a mock with verbose logging
2. reset a mock

What is the expected output? What do you see instead?
Verbose logging doesn't work after reseting a mock.

What version of the product are you using? On what operating system?
1.9.0-rc1

Patch attached.

```

Original issue reported on code.google.com by `dpredo...@gmail.com` on 30 Sep 2011 at 5:52

Attachments:
- [reset.patch](https://storage.googleapis.com/google-code-attachments/mockito/issue-282/comment-0/reset.patch)

**Comment by GoogleCodeExporter** [comment]
```
Cool, thx for the test and the patch.
```

Original comment by `brice.du...@gmail.com` on 1 Oct 2011 at 2:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `brice.du...@gmail.com` on 3 Oct 2011 at 8:12
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `szcze...@gmail.com` on 16 Dec 2011 at 9:42
- Added labels: **Milestone-Release1.9.0**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revision b2390bbba786
```

Original comment by `brice.du...@gmail.com` on 16 Dec 2011 at 10:28
